mixes/reducer.js

```diff
@@ -29,10 +34,12 @@ function createReducer (config) {
       Effects.promise(runLoadMixList),
       Effects.constant(loadMixListEnd())
     ])),
-    [loadMixListSuccess]: (state, action) => ({
-      ...state,
-      records: merge({}, state.records, keyBy(action.payload, 'id'))
-    }),
+
+    // how to get this to not run LOAD_MIX_END end until all SET_MIX are done?
```

Comment:
hmm well their example has it the way we have it now. leaving as is since this can be easily refactored later